api/swagger.yaml

```diff
@@ -1923,6 +1923,38 @@ definitions:
                 type: "array"
                 items:
                   type: "string"
+          Secrets:
+            description: "Secrets contains references to zero or more secrets that will be exposed to the service."
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                File:
```

Comment:
@ehazlett Currently the naming is in:
https://github.com/docker/docker/blob/v1.13.1/api/types/swarm/secret.go#L19-L31
I think a deprecation might be needed if it is changed.